Cargo.toml

```diff
@@ -26,7 +26,7 @@ lazy_static = "1.4.0"
 encode_unicode = "0.3"
 
 [target.'cfg(windows)'.dependencies.windows-sys]
-version = "0.52.0"
+version = "0.59.0"
```

Comment:
I would suggest using something like `>=0.52.0, <=0.59.*` for this if possible. Or even `>=0.48.0,<=0.59.*`. This will give Cargo a bit more flexibility and avoid including multiple versions of `windows-sys` in the dependency tree. [This is what I'm doing in `winapi-util`.](https://github.com/BurntSushi/winapi-util/pull/19) I'm [also doing this in `jiff`](https://github.com/BurntSushi/jiff/blob/ce9c60ee4659d905dce6d2f31ad04930add4af14/Cargo.toml#L89-L93).